{"uid":"41ea0ac3b958bb09","name":"test_relations","fullName":"tests.integration.new_relations.test_relations_coherence#test_relations","historyId":"bed3ea11af505ce0051869405361cb26","time":{"start":1728351045871,"stop":1728351175316,"duration":129445},"description":"Test that check relation data.","descriptionHtml":"<p>Test that check relation data.</p>\n","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/new_relations/test_relations_coherence.py\", line 35, in test_relations\n    await ops_test.model.wait_for_idle(apps=[DATABASE_APP_NAME], status=\"active\", timeout=3000)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/0","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1728351045649,"stop":1728351045650,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1728351045650,"stop":1728351045650,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1728351045650,"stop":1728351045868,"duration":218},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1728351045868,"stop":1728351045870,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1728351045870,"stop":1728351045870,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test that check relation data.","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/new_relations/test_relations_coherence.py\", line 35, in test_relations\n    await ops_test.model.wait_for_idle(apps=[DATABASE_APP_NAME], status=\"active\", timeout=3000)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/0","steps":[],"attachments":[{"uid":"264ba6a2bd0dda70","name":"log","source":"264ba6a2bd0dda70.txt","type":"text/plain","size":2393}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1728351175326,"stop":1728351175326,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm::finalizer","time":{"start":1728351175326,"stop":1728351175326,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1728351175327,"stop":1728351175710,"duration":383},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1728351175710,"stop":1728351175710,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1728351175710,"stop":1728351175710,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1728351175710,"stop":1728351175711,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.new_relations"},{"name":"suite","value":"test_relations_coherence"},{"name":"host","value":"runner"},{"name":"thread","value":"17588-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.new_relations.test_relations_coherence"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"c0b31e974c6f73d7","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":22,"unknown":0,"total":23},"items":[{"uid":"11928c973ee6c7ed","reportUrl":"../103//#testresult/11928c973ee6c7ed","status":"passed","time":{"start":1728264795700,"stop":1728265221338,"duration":425638}},{"uid":"45cd349d834389a3","reportUrl":"../102//#testresult/45cd349d834389a3","status":"passed","time":{"start":1728178328475,"stop":1728178755276,"duration":426801}},{"uid":"e99208f0a0391c88","reportUrl":"../101//#testresult/e99208f0a0391c88","status":"passed","time":{"start":1728091843133,"stop":1728092267117,"duration":423984}},{"uid":"dd6c5f98ff90e2f2","reportUrl":"../100//#testresult/dd6c5f98ff90e2f2","status":"passed","time":{"start":1728005519605,"stop":1728005935140,"duration":415535}},{"uid":"44dae930b3972ffa","reportUrl":"../99//#testresult/44dae930b3972ffa","status":"passed","time":{"start":1727919120988,"stop":1727919544006,"duration":423018}},{"uid":"34308808c5073614","reportUrl":"../98//#testresult/34308808c5073614","status":"passed","time":{"start":1727832699791,"stop":1727833121453,"duration":421662}},{"uid":"36c55ad583aefb7a","reportUrl":"../97//#testresult/36c55ad583aefb7a","status":"passed","time":{"start":1727746333296,"stop":1727746756040,"duration":422744}},{"uid":"89f47e61c48d77aa","reportUrl":"../96//#testresult/89f47e61c48d77aa","status":"passed","time":{"start":1727659907553,"stop":1727660343131,"duration":435578}},{"uid":"96dca66808a8069a","reportUrl":"../95//#testresult/96dca66808a8069a","status":"passed","time":{"start":1727573502559,"stop":1727573929177,"duration":426618}},{"uid":"a6b9eabf9598f533","reportUrl":"../94//#testresult/a6b9eabf9598f533","status":"passed","time":{"start":1727487077342,"stop":1727487503570,"duration":426228}},{"uid":"f8ee7e1089563896","reportUrl":"../93//#testresult/f8ee7e1089563896","status":"passed","time":{"start":1727400754402,"stop":1727401178041,"duration":423639}},{"uid":"64610821b3b40dfe","reportUrl":"../92//#testresult/64610821b3b40dfe","status":"passed","time":{"start":1727314449560,"stop":1727314875039,"duration":425479}},{"uid":"6e4d6cbdc79d84be","reportUrl":"../90//#testresult/6e4d6cbdc79d84be","status":"passed","time":{"start":1722217068652,"stop":1722217450631,"duration":381979}},{"uid":"853847ce60ca5542","reportUrl":"../89//#testresult/853847ce60ca5542","status":"passed","time":{"start":1722130490964,"stop":1722130811633,"duration":320669}},{"uid":"cfe055f509f1a862","reportUrl":"../88//#testresult/cfe055f509f1a862","status":"passed","time":{"start":1722044192985,"stop":1722044545420,"duration":352435}},{"uid":"3bb97105afb0cee5","reportUrl":"../87//#testresult/3bb97105afb0cee5","status":"passed","time":{"start":1721957581869,"stop":1721957900085,"duration":318216}},{"uid":"e9fc5e2b34f0ab4a","reportUrl":"../86//#testresult/e9fc5e2b34f0ab4a","status":"passed","time":{"start":1721871305672,"stop":1721871654747,"duration":349075}},{"uid":"58e70f33183914ba","reportUrl":"../85//#testresult/58e70f33183914ba","status":"passed","time":{"start":1721785015356,"stop":1721785413897,"duration":398541}},{"uid":"c96dd65d810b23b7","reportUrl":"../84//#testresult/c96dd65d810b23b7","status":"passed","time":{"start":1721698489557,"stop":1721698841645,"duration":352088}},{"uid":"788e96871ff84aa","reportUrl":"../83//#testresult/788e96871ff84aa","status":"passed","time":{"start":1721612526137,"stop":1721612867554,"duration":341417}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"41ea0ac3b958bb09.json","parameterValues":[]}